index.js

```diff
@@ -25,7 +25,7 @@ module.exports = (tag, opts) => {
 };
 
 function * main(tag, opts) {
-	const pixiv = new Pixiv(opts.username, opts.password);
+	const pixiv = new Pixiv(opts.username.toString(), opts.password.toString());
```

Comment:
My password is a number,while `Meow` wont convert it to a string which `PixivAuthGot` needs.